python/ray/data/_internal/execution/operators/one_to_one_operator.py

```diff
@@ -10,7 +10,31 @@
 from ray.types import ObjectRef
 
 
-class LimitOperator(PhysicalOperator):
+class OneToOneOperator(PhysicalOperator):
+    """A streaming operator that executes once its input is ready.
+
+    This operator serves as the base for map, filter, limit, etc.
+    """
+
+    def __init__(
+        self,
+        input_op: PhysicalOperator,
+        name: str = "OneToOne",
```

Comment:
Don't set the default value for name. Because the subclasses are supposed to specify the name.